DataTool/Static/GUIDNames.csv

```diff
@@ -1,4 +1,4 @@
-﻿GUID Index, GUID Type, Name, Canonical
```

Comment:
There was a zwnbsp here apparently cattes

Reply:
that wasn't a zwnbsp, it was a UTF-8 BOM.

not needed anyway just some stuff windows adds sometimes.